cmd/jujud-controller/main_debug.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2012, 2013 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+//go:build debug
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"slices"
+
+	"github.com/juju/juju/internal/dlv"
+)
+
+func main() {
+	args := os.Args
+
+	// enable debug except for command version and bootstrap-state which rely on command output. Debugging server will
```

Comment:
``Enable debug except for `version` and `bootstrap-state` commands, which...``

Reply:
Done https://github.com/juju/juju/pull/18114/commits/8735fc258e9495c527885ea569ef7368901991d6